tensorflow_addons/losses/lifted.py

```diff
@@ -17,10 +17,16 @@
 import tensorflow as tf
 from tensorflow_addons.losses import metric_learning
 
+from tensorflow_addons.utils.types import FloatTensorLike, Number, TensorLike
+from typeguard import typechecked
+from typing import Optional
+
 
 @tf.keras.utils.register_keras_serializable(package="Addons")
 @tf.function
-def lifted_struct_loss(labels, embeddings, margin=1.0):
+def lifted_struct_loss(
+    labels: FloatTensorLike, embeddings: TensorLike, margin: Number = 1.0
```

Comment:
```suggestion
    labels: TensorLike, embeddings: TensorLike, margin: Number = 1.0
```

I really need to change the name to ScalarTensor.